pandas-stubs/core/frame.pyi

```diff
@@ -1874,7 +1874,7 @@ class DataFrame(NDFrame, OpsMixin):
     def pct_change(
         self,
         periods: int = ...,
-        fill_method: _str = ...,
+        fill_method: _str | None = ...,
```

Comment:
Since the only acceptable argument is now `None` because the string values have been deprecated, can you just change this to 
```suggestion
        fill_method:  None = ...,
```

Reply:
Wasn't sure how far the support of pandas-stubs would go so I did not want to go too hard but agree with the deprecation!